project_task_pull_request_state/views/project_task_pull_request_state_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="project_view_task_form_pr_state" model="ir.ui.view">
```

Comment:
Record id of the inheriting view should be the same as the inherited one.
See [OCA guidlines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#253inherited-xml)